src/Bitcoin/Script.cpp

```diff
@@ -252,21 +252,18 @@ void Script::encode(std::vector<uint8_t>& data) const {
     std::copy(std::begin(bytes), std::end(bytes), std::back_inserter(data));
 }
 
-Script Script::buildForAddress(const std::string& string) {
+Script Script::buildForAddress(const std::string& string, enum TWCoinType coin) {
     if (Address::isValid(string)) {
-        auto p2pkhPrefixes = TW::p2pkhPrefixes();
```

Comment:
do you think it might be a good idea to split this logic later to have it implemented in scripts for each compatible blockchain?

Reply:
would be too many changes, we can split it for new chains